MAUI/TreeMap/events.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: Events in .NET MAUI TreeMap (SfTreemap) control | Syncfusion
+description: Learn here all about the Events support in the Syncfusion .NET MAUI TreeMap (SfTreemap) control and more.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+
+# Events in the .NET MAUI TreeMap (SfTreeMap)
+
+## SelectionChanged
+
+The `SelectionChanged` event occurs when there's a change in the selection within a tree map item.
+Below is a list of the arguments:
+
+* `Sender`: This contains the `SfTreeMap` object.
```

Comment:
`Sender`: This contains the `SfTreeMap` object.

No need to add this

Reply:
Removed